test/indexOf_test.js

```diff
@@ -4,9 +4,13 @@ const chaiChange = require('chai-change')
 chai.use(chaiChange)
 
 const indexOf = require('../src/indexOf')
-describe('indexOf()'), () => {
+describe('indexOf()', () => {
 
-  it('gets index of passed in value', () => {
+   it('gets index of passed in value', () => {
     expect( indexOf([1, 2, 1, 2], 2)).to.equal(1)
-  })
+   it('gets index of passed in value', () => {
+    expect( indexOf([1, 2, 1, 2], 2)).to.equal(1)
```

Comment:
saw or test was repeated, that wasnt intentional here is the right second test
it('gets index of passed in value', () => {
    expect( indexOf([1, 2, 1, 2], 2, 2)).to.equal(3)